test/e2e/network/service.go

```diff
@@ -2309,6 +2311,466 @@ var _ = common.SIGDescribe("Services", func() {
 		}
 	})
 
+	ginkgo.It("should fail health check node port if there are only terminating endpoints [Feature:ProxyTerminatingEndpoints]", func() {
```

Comment:
```suggestion
	ginkgo.It("should fail health check node port if there are only terminating endpoints [Feature:ProxyTerminatingEndpoints][LinuxOnly]", func() {
```

Reply:
People do this all over, but I feel like `[LinuxOnly]` should only be used to mean "this uses features that can't be implemented on Windows" not "this uses features that happen to not currently be implemented on Windows". (But it's possible there are official guidelines that say otherwise...)

Reply:
Yeah agreed with Dan, I didn't put LinuxOnly because there's no kernel specific dependency for this feature, we just didn't implement it on windows yet

Reply:
but do they work in windows?
This is a sig-testing thing, the convention is to use tags for filtering, if it only works in Linux, they should use that tag so we can filter them ...

Reply:
That's what I was saying about maybe the guidelines say otherwise. But FTR, that's a stupid convention. Windows should keep a list of the features they don't implement (like `[Feature:ProxyTerminatingEndpoints]` and the tests can filter on that. Rather than requiring every test case to know the current status of Windows support.